tests/logout.test.js

```diff
@@ -0,0 +1,37 @@
+import LoginPage from '../pages/login.page';
+import LogoutPage from '../pages/logout.page';
+import HomePage from '../pages/home.page';
+import BasePage from '../pages/base.page';
+
+const LOGIN_PAGE = LoginPage;
+const BASE_PAGE = BasePage;
+
+const VALID_USERNAME = 'jsmith@demo.io';
+const ANOTHER_VALID_USERNAME = 'nsmith@demo.io';
+const VALID_PASSWORD = 'Demo123!';
+
+
+describe('Logout Test Cases', () => {
+  beforeEach(async function  () {
+    await LOGIN_PAGE.navigateToPage(); 
+  });
+
+  it('Should logout successful when click on the Logout button', async () => {
+    await LOGIN_PAGE.logInWithSubmitButton(VALID_USERNAME, VALID_PASSWORD);
+    await BASE_PAGE.clickElement(LogoutPage.profileIcon);
```

Comment:
After you make LoginPage inherit from BasePage, you can call this method directly on LoginPage. Make sure to change this call to that after you implement inheritance